src/testsuite/testsuite.jl

```diff
@@ -5,7 +5,19 @@ Abstract test suite that can be used for all packages inheriting from GPUArray
 """
 module TestSuite
 
-using Base.Test
+using GPUArrays
+using GPUArrays: mapidx, gpu_sub2ind
+
+if GPUArrays.is_v07
```

Comment:
FWIW, this kind of thing is not as good as a proper version check, since femtocleaner won't remove it.